metadata/src/main/java/edu/unc/lib/dl/rdf/Premis.java

```diff
@@ -192,7 +195,7 @@ public static String getURI() {
             "http://www.loc.gov/premis/rdf/v1#TermOfRestriction");
 
     /** Premis event types
```

Comment:
nitpick, but maybe move comment down one line from the "/**"

Reply:
That was one of the weird things that the rdf schema to java class tool generated, i'll fix it though